architecture/adr-list/adr-002-identity-transactions.md

```diff
@@ -0,0 +1,272 @@
+- **Namespace Identifier**: A self-certified identifier unique to the given cheqd DID ledger namespace. To be self-certifying, the identifier must be derived from the initial `verkey` associated with the identifier.
+
+The components are assembled as follows:
+
+`did:cheqd:<namespace>:<namespace identifier>`
+
+Some examples of `did:cheqd` method identifiers are:
+
+- A DID written to the cheqd mainnet ledger: `did:cheqd:mainnet:7Tqg6BwSSWapxgUDm9KKgg`
+- A DID written to the cheqd testnet ledger: `did:cheqd:testnet:6cgbu8ZPoWTnR5Rv5JcSMB`
+
+### General structure of transaction requests
+
+All identity requests will have the following format:
+
+```javascript
+{
+    "data": { "<request data for writing a transaction to the ledger>" },
+    "creators": ["<identifier>", "..."],
+    "signatures": [
+        "<public_key>": "<signature>",
+      ],
+    "metadata": {}
+}
+```
+
+- **`data`**: Data requested to be written to the ledger, specific for each request type.
+- **`creators`**: Creators identifiers (DID) list for this entity. There should be a new DIDs or an existing DIDs, for existing entities.
+- **`signatures`**: `data` should be signed by all `creators` private key. This field contains a dict there creator's public key is a key, and the signature is a value.
+- **`metadata`**: Dictionary with additional metadata fields. Empty for now. This fields provides extensibility in the future, e.g., it can contain `protocolVersion` or other relevant metadata associated with a request.
+
+## List of transactions and details
+
+### `DID` transactions
+
+[Decentralized Identifiers \(DIDs\) are a W3C specification](https://www.w3.org/TR/did-core/) for identifiers that enable verifiable, decentralized digital identity.
+
+DIDDoc format conforms to [DIDDoc spec](https://www.w3.org/TR/did-core/#representations).
+The request can be used for creation of new DIDDoc, setting, and rotation of verification key.
+
+#### DIDDoc
+
+1. **`id`**: Target DID as base58-encoded string for 16 or 32 byte DID value. In the ledger, we store only an identifier without specifying the method and namespace.
+2. **`controller`** (optional): A list of base58-encoded identifier strings.
+3. **`verificationMethod`** (optional): A list of Verification Methods
+4. **`authentication`** (optional): A list of Verification Methods or strings with key aliases
+5. **`assertionMethod`** (optional): A list of Verification Methods or strings with key aliases
+6. **`capabilityInvocation`** (optional): A list of Verification Methods or strings with key aliases
+7. **`capabilityDelegation`** (optional): A list of Verification Methods or strings with key aliases
+8. **`service`** (optional): A set of Service Endpoint maps
+9. **`@context`** (optional): A list of strings
```

Comment:
This has a relatively stable definition within DID spec as it typically refers back to W3C DID standards, and can therefore be referred to as such.

Reply:
I extended a description here. But the format of this field is flexible in DIDDoc. A list of any strings provides it.